src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaListExtensions.cs

```diff
@@ -47,6 +48,34 @@ public static IEnumerable<SyntaxTrivia> SkipInitialWhitespace(this SyntaxTriviaL
             return triviaList.SkipWhile(t => t.Kind() == SyntaxKind.WhitespaceTrivia);
         }
 
+        private static IEnumerable<List<SyntaxTrivia>> GetLines(this SyntaxTriviaList triviaList)
```

Comment:
1. Consider making into `ImmutablArray<ImmutableArray<SyntaxTrivia>>`.    
2. Consider renaming into `BreakIntoLines`.

Reply:
This method worries me as the name doesn't quite convey what it is.  For example, if i have a multiline comment, i won't get it as multiple lines.